android/src/main/java/com/rnfs/RNFSManager.java

```diff
@@ -75,15 +123,13 @@ public void writeFile(String filepath, String base64Content, ReadableMap options
   @ReactMethod
   public void readFile(String filepath, Callback callback) {
     try {
-      File file = new File(filepath);
-
-      if (!file.exists()) throw new Exception("File does not exist");
```

Comment:
Can you add please again the File-Exists-check and throw an Exception, if the file does not exists? To not throw an exception would be compatibility break for all those that rely on this test (instead of calling .exists() by themself in JS-layer)

Reply:
It actually doesn't start with `file://` since the uri scheme is null